app/code/Magento/Cms/Ui/Component/Listing/Column/BlockActions.php

```diff
@@ -32,24 +32,27 @@ class BlockActions extends Column
     /**
      * @var Escaper
      */
-    private $escaper;
+    protected $escaper;
```

Comment:
What's the reason for changing this? I don't immediately see why this would be needed? I think this can be put back to `private`, no?

Same remark goes for the other class.

Reply:
@hostep  I'm agree with you, but at the same time I don't see any restrictions to use protected access type

Reply:
Purely because the UrlBuilder was protected as well. I thought that is some kind of coding standard.

I'd be happy to change it to private if you agree to this change.

Reply:
@Den4ik: Magento encourages composition over inheritance, if we make members protected again, we are pushing people in the direction of using inheritance again.

I would suggest we change it back to `private`.

Reply:
@hostep It make sense 👍

Reply:
@hostep Great. I updated the files accordingly. Please review.